install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -1015,14 +1015,14 @@ data:
 
 {{- end }}
 
-{{- if .Values.auth.mTLS.enabled }}
+{{- if .Values.auth.mTLS.spire.enabled }}
   mesh-auth-mtls-enabled: "true"
   mesh-auth-mtls-listener-port: {{ .Values.auth.mTLS.port | quote }}
-  mesh-auth-spire-agent-socket: {{ .Values.auth.mTLS.spireAgentSocketPath }}
-  mesh-auth-spire-server-address: {{ .Values.auth.mTLS.spireServerAddress }}
-  mesh-auth-spire-server-connection-timeout: {{ .Values.auth.mTLS.spireServerConnectionTimeout }}
-  mesh-auth-spire-admin-socket: {{ .Values.auth.mTLS.spireAdminSocketPath }}
-  mesh-auth-spiffe-trust-domain: {{ .Values.auth.mTLS.spiffeTrustDomain }}
+  mesh-auth-spire-agent-socket: {{ .Values.auth.mTLS.spire.agentSocketPath | quote }}
+  mesh-auth-spire-server-address: "spire-server.{{ .Values.auth.mTLS.spire.install.namespace }}.svc.cluster.local:8081"
```

Comment:
the .cluster.local can bite you when you change `--cluster-domain=` in your k8s setup

Reply:
good point :100:.

I Just double check in other places, we leave it to user to supply value. This could ease the path for BYO spire later as well.